docs/api.md

```diff
@@ -151,19 +151,15 @@ resource.
 
 ### Parameters
 
-Name | Type | Required | Description
-:--: | :--: | :------:| -----------
-resource | String | Yes | The resource to query
-props| Array | No | Specifies the desired properties to be returned in each record.  An empty array or null value means all properties will be returned.
-where | Object | No | Specifies constraints to filter the data.  Null means all records will be returned.
-sort | Object | No | Specifies the desired sort for the result set.
+**`String resource`**:  (*Required*) The name of the resource type to query.
+ Must be either `SystemResources.TOPICS.value()`, `SystemResources.SOURCES.value()` or `SystemResources.TYPES.value()`.
 
-The `props` is an array of property names indciating which properties should be returned.
-    
-The `where` is an object with supported operations defined in [API operations](https://dev.vantiq.com/docs/system/api/index.html#where-parameter).
+**`List<String> props`**: (*Required*) Specifies the desired properties to be returned in each record.  An empty list or null value means all properties will be returned.
+
+**`Object where`**: Specifies constraints to filter the data.  Null means all records will be returned.
+
+**`SortSpec sort`**: Specifies the desired sort for the result set
```

Comment:
It looks like you just copied the Java SDK docs here, but I don't think you should. for one thing, the parameter classes don't match, there are no generics in javascript, and there's no `SortSpec` class.